ui/app/src/components/PreviewInPageInstancesPanel/PreviewInPageInstancesPanel.tsx

```diff
@@ -95,6 +93,8 @@ export function PreviewInPageInstancesPanel() {
   const models = useMemo(() => {
     return guest?.models;
   }, [guest]);
+  const filteredContentTypes =
+    models && Object.values(models)?.filter((model) => model.craftercms.contentTypeId === contentTypeFilter);
```

Comment:
I don't think Object.values would ever result in null.
```suggestion
    Object.values(models ?? {}).filter((model) => model.craftercms.contentTypeId === contentTypeFilter);
```